app/services/hyrax/custom_queries/find_access_control.rb

```diff
@@ -19,6 +19,8 @@ def find_access_control_for(resource:)
           .find_inverse_references_by(resource: resource, property: :access_to)
           .find { |r| r.is_a?(Hyrax::AccessControl) } ||
           raise(Valkyrie::Persistence::ObjectNotFoundError)
+      rescue ArgumentError # some adapters raise ArgumentError for missing resources
```

Comment:
Oof…that sounds like an upstream inconsistency. @tpendragon what are your thoughts as this relates to Valkyrie?

Reply:
Via `find`? They definitely should not raise `ArgumentError`, and that'd be a bug. I'm pretty certain it's even in the shared_specs.

Reply:
via `.find_inverse_references_by`

Reply:
@tpendragon or @no-reply Any further thoughts/discussions/necessary issues to submit?

Reply:
@no-reply Could you make a ticket in Valkyrie with the details? `find_inverse_references_by` should be returning an empty array.